src/store.js

```diff
@@ -1,10 +1,12 @@
 import { configureStore } from '@reduxjs/toolkit';
 import { reducer as mainReducer } from 'features/Main/data';
+import { reducer as commonReducer } from 'features/Common/data';
 
 export function initializeStore(preloadedState = undefined) {
   return configureStore({
     reducer: {
       main: mainReducer,
+      common: commonReducer,
```

Comment:
Common? Why not `classes`?

Reply:
Not all the services will have a page as IP, an as this data will be used in different pages is a common data